src/content/glossary/index.md

```diff
@@ -123,6 +123,14 @@ A group of at least 128 [validators](#validator) assigned to beacon and shard bl
 
 When numerous nodes (usually most nodes on the network) all have the same blocks in their locally validated best blockchain. Not to be confused with [consensus rules](#consensus-rules).
 
+### consensus client {consensus-client}
+
+After the merge to [proof-of-stake](#pos), consensus about the true head of the blockchain will be governed by a new set of clients running on the Beacon Chain known as "consensus clients". Consensus clients do not participate in validating transactions or executing state transitions. They request this to be done by [execution clients](#execution-client).
```

Comment:
> After the merge to [proof-of-stake](#pos), consensus about the true head of the blockchain will be governed by a new set of clients running on the Beacon Chain known as "consensus clients".

While this is true, I would add a sentence before to explain what consensus clients are before going into The Merge, given they exist today. Perhaps something like "Consensus clients (a.k.a. Beacon Chain client) run Ethereum's proof of stake consensus algorithm." I would also suggest naming a few of them to make it clear to readers "what" they are. 

Also a few small nits:
- "the merge to proof of stake" -> "the transition to proof of stake" OR "the merge": we are moving from PoW to PoS *by* merging the changes, so it's a bit odd to say "the merge to PoS";
- "true head" -> "head": by saying you are coming to consensus on it, you imply it's "true";
- "will be governed" -> "will be determined": there isn't really governance involved here, risks confusing people;
- "a new set of clients": not quite accurate, given they are already existent. Assuming you added the earlier sentence I proposed, you can just use "these clients". 


> Consensus clients do not participate in validating transactions or executing state transitions. They request this to be done by [execution clients](#execution-client).

Two small nits here: 

- "validating" -> "validating/broadcasting": txn gossip will be on the EL;
- "They request this to be done" -> "This is done by": while it's technically true the consensus client makes a request to the execution layer client, this phrasing implies some sort of superiority, which I think we want to avoid.